test/payment/eip712/index.js

```diff
@@ -0,0 +1,4 @@
+const ApproveForwarder = require('./ApproveForwarder.eip712');
```

Comment:
recommend to move this file somewhere else than in payment folder as it is only related to meta-tx